### PR TITLE
Add missing quote

### PR DESCRIPTION
**Pull request by shrujalshah28**

### CODE REVIEW

File: lib/msal-angular/README.md [hunk 1]
```diff
@@ -161,7 +161,7 @@ Defaults to `window.location.href`.
 
 * **validateAuthority** : Validate the issuer of tokens. Default is true.
 
-* **cacheLocation** : Sets browser storage to either 'localStorage' or sessionStorage'. Defaults is 'sessionStorage'.
+* **cacheLocation** : Sets browser storage to either 'localStorage' or 'sessionStorage'. Defaults is 'sessionStorage'.
```
Comment (jasonnutter) [inline lib/msal-angular/README.md]: You can also make these backticks so they get formatted nicely:

```md
* **cacheLocation** : Sets browser storage to either `localStorage` or `sessionStorage`. Defaults is `sessionStorage`.
```
* **cacheLocation** : Sets browser storage to either `localStorage` or `sessionStorage`. Defaults is `sessionStorage`.

Comment (sameerag) [reply lib/msal-angular/README.md]: Defaults **to** sessionStorage

**Comment by DarylThayil** [comment]
@shrujalshah28  were you still planning to make this update?

**Comment by sameerag** [comment]
@DarylThayil I think he already addressed the comments. I just approved.

**Comment by DarylThayil** [comment]
@jasonnutter  @pkanher617  can you resolve your changes request?

**Comment by sameerag** [comment]
@pkanher617 Is this approvable now? We can merge if you are alright.

**Pull request merged**